pom.xml

```diff
@@ -0,0 +1,118 @@
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
Should this pom.xml file be the same as the one I'm working with in backend/ ?

Reply:
+1

Reply:
I think we're all going to be using the same pom.xml, I had moved it previously just to test my maven. If the files are override and some dependencies go missing i think we can ultimately resolve them